stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -547,7 +550,10 @@ extension TFState {
       targetNodeSpecs, Int32(targetNodeSpecs.count),
       /*run_metadata*/nil, status
     )
-    checkOk(status)
+    if (TF_GetCode(status) != TF_OK) {
+      logToStderr(String(cString: TF_Message(status)))
```

Comment:
It doesn't matter much, but calling `fputs` is easier and more efficient.

```swift
_ = fputs(TF_Message(status), stderr)
```

Reply:
Done.